templates/_pagination.html

```diff
@@ -39,6 +39,17 @@
 					</a>
 				{% endif %}
 			</li>
+			<li>
+				<form class="navbar-form navbar-left" action="." method="get">
+					<div class="form-group">
+						<select name="paginate_by" class="paginate_by" onchange="this.form.submit()">
```

Comment:
If the teacher has filtered feedback in the filter form, then those filters are lost from the URL GET query parameters when this `paginate_by` select menu triggers the form submission. The user is unable to filter feedback and also change the page size. The only way currently for the teacher to do both is to manually modify the URL query parameters, but that is bad for usability.

I think the `onchange` attribute must be removed in `<select onchange="...">`. We need to implement an event handler in a separate JS script. That script could probably avoid submitting the form. Instead, we can redirect the user to the new paginated results by just setting `window.location`. In that case, we can keep all the old GET query parameters and also modify the `paginate_by` parameter.